ui-v2/tests/acceptance/steps/dc/acls/index-steps.js

```diff
@@ -0,0 +1,10 @@
+import steps from '../../steps';
+
+// step definitions that are shared between features should be moved to the
+// tests/acceptance/steps/steps.js file
+
+export default function(assert) {

```

Comment:
Kinda a bummer to have all of these, but not that big a deal in the end ¯\_(ツ)_/¯

Reply:
Defo, I did hack into the addon to have a look, and its easy enough to not require these, so needs a PR to the addon, probably look at that in a few weeks. I can't remember whether I looked to see how way it would be for to specify `--no-steps` or something for the generator.